.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -47,12 +49,22 @@ Push access to the upstream repository is required in order to publish the new t
   - [ ] `git push ${UPSTREAM_REMOTE} v${RELEASE_VER}`
   - [ ] `cargo publish`
 
+- assemble vendor archive:
+  - [ ] `cargo vendor`
```

Comment:
Could we `cargo vendor target/vendor` instead so then `cargo clean` would clean up the vendor artifacts as well (so we don't need `rm -rf vendor`)?